pkg/sypgp/sypgp.go

```diff
@@ -515,20 +516,20 @@ func GenKeyPair(keyServiceURI string, authToken string) (entity *openpgp.Entity,
 
 // DecryptKey decrypts a private key provided a pass phrase
 func DecryptKey(k *openpgp.Entity, message string) error {
+	if !k.PrivateKey.Encrypted {
+		return errNotEncrypted
+	}
```

Comment:
I still honestly think this should not return a err, but thats my opinion and maybe I'm wrong.